CDP4Composition/DragDrop/TreeListSelectionStrategyRowDragDrop.cs

```diff
@@ -0,0 +1,76 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="TreeListSelectionStrategyRowDragDrop.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2022 RHEA System S.A.
+// 
+//    Author: Sam Gerené, Alex Vorobiev, Naron Phou, Patxi Ozkoidi, Alexander van Delft, Nathanael Smiechowski, Ahmed Ahmed, Simon Wood
+// 
+//    This file is part of COMET-IME Community Edition.
+//    The COMET-IME Community Edition is the RHEA Concurrent Design Desktop Application and Excel Integration
+//    compliant with ECSS-E-TM-10-25 Annex A and Annex C.
+// 
+//    The COMET-IME Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Affero General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or any later version.
+// 
+//    The COMET-IME Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+//    Lesser General Public License for more details.
+// 
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4Composition.DragDrop
+{
+    using System.Windows;
+    using System.Windows.Input;
+
+    using DevExpress.Xpf.Core;
+    using DevExpress.Xpf.Grid;
+    using DevExpress.Xpf.Grid.TreeList;
+
+    /// <summary>
+    /// Extended row selection strategy that override mouse down and up behaviors when using Ctrl key
+    /// </summary>
+    public class TreeListSelectionStrategyRowDragDrop : TreeListSelectionStrategyRow
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TreeListSelectionStrategyRowDragDrop"/> class
+        /// </summary>
+        /// <param name="view">The tre view</param>
+        public TreeListSelectionStrategyRowDragDrop(TreeListView view) : base(view)
+        {
+        }
+
+        /// <summary>
+        /// Override of the Left mouse button down behavior
+        /// </summary>
+        /// <param name="hitInfo">The hitinfo</param>
+        protected override void OnAfterMouseLeftButtonDownCore(IDataViewHitInfo hitInfo)
+        {
+            if (Keyboard2.IsControlPressed)
```

Comment:
what is `keyboard2`? can we use a different name without the `2`?

Reply:
Its a static class in the DevExpress that is advised to be used, can't change the name unfortunately.